x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/ChoroplethMap/index.tsx

```diff
@@ -263,16 +247,21 @@ export const ChoroplethMap: React.SFC<Props> = props => {
     }
   }, [map, tooltipState]);
 
+  const toolTipProps = { name: '', value: 0, docCount: 0, ...tooltipState };
+  const hasMinMax = Number.isFinite(min) && Number.isFinite(max);
+
   // render map container and tooltip in a hidden container
   return (
     <div>
       <div ref={containerRef} style={{ height: 256 }} />
       <div style={{ display: 'none' }}>
         <div ref={popupContainerRef}>
-          <ChoroplethToolTip {...tooltipState} />
+          <ChoroplethToolTip {...toolTipProps} />
```

Comment:
Instead of setting default values, what about not showing the tooltip if there is no tooltip data to show?
```js
{tooltipState && <ChoroplethToolTip {... tooltipState} />}
```